README.md

```diff
@@ -109,6 +110,20 @@ $ nrfjprog --program feather_nrf52832_bootloader.hex -f nrf52
 $ nrfjprog --reset -f nrf52
 ```
 
+## Particle Xenon Specific
+```c++
+// Swap between the internal antenna and external u.FL antenna
+switchAntenna(bool useExternal);
```

Comment:
Please remove these all xenon-specific functions, user should be able to do that in user sketch